src/Combinatorics/Matroids/properties.jl

```diff
@@ -1030,6 +1030,58 @@ function min_revlex_basis_encoding(M::Matroid)
     return _revlex_basis_from_vector(min_rvlx)
 end
 
+@doc raw"""
+    matroid_hex(M::Matroid)
+
+Stores a matroid as a string of hex characters. The first part of the string is "r" followed by the rank of the matroid. This is followed by "n" and the number of elements. The rest of the string is the revlex basis encoding. The encoding is done by converting the basis encoding to a vector of bits and then to a string of characters. The bits are padded to a multiple of 4 and then converted to hex characters.
+
+# Examples
+To get the hex encoding of the fano matroid write:
+```jldoctest
+julia> matroid_hex(fano_matroid())
+"r3n7_3f7eefd6f"
+
+```
+"""
+function matroid_hex(M::Matroid)::String
+  rvlx = min_revlex_basis_encoding(M)
+  r,n = rank(M), length(M) 
+
+  v = _revlex_basis_to_vector(rvlx)
+  for _ in 1:(4-length(v)%4)
+    pushfirst!(v,0)
+  end
+  v = reshape(v,4,:)
+  v = [string(parse(Int,join(v[:,j]),base=2),base=16) for j in 1:size(v)[2]]
```

Comment:
you should in general be a little more generous with your spaces, see https://docs.oscar-system.org/stable/DeveloperDocumentation/styleguide/#Whitespace
```suggestion
  v = [string(parse(Int, join(v[:, j]), base=2), base=16) for j in 1:size(v)[2]]
```